src/NzbDrone.Core/Indexers/Definitions/Avistaz/AvistazParser.cs

```diff
@@ -61,7 +58,7 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                     InfoUrl = details,
                     Guid = details,
                     Categories = cats,
-                    PublishDate = DateTime.Parse(row.CreatedAt + "-05:00").ToUniversalTime(), // Avistaz does not specify a timezone & returns server time
+                    PublishDate = DateTime.Parse($"{row.CreatedAt} {TimezoneOffset}").ToUniversalTime(),
```

Comment:
Should we convert to use `DateTimeStyles.AdjustToUniversal` in the parse. Docs seem to indicate it's a bit more efficient


> The same effect can be achieved by calling the [DateTime.ToUniversalTime](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.touniversaltime?view=net-7.0) or [DateTimeOffset.ToUniversalTime](https://learn.microsoft.com/en-us/dotnet/api/system.datetimeoffset.touniversaltime?view=net-7.0) method. However, using the [AdjustToUniversal](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.datetimestyles?view=net-7.0#system-globalization-datetimestyles-adjusttouniversal) value with the date and time parsing method is more efficient.